### DISCUSSION TITLE
Making Decap CMS reponsive and render on mobile

### DISCUSSION THREAD

**Discussion by privatemaker** [decaporg/decap-cms#7083] [Show and tell]
I'm just posting something mentioned in Discord (for those who are not there) which is useful and was created by @hithismani 🙏🏻

- [Forcing Mobile Responsive Support for Decap/Netlify CMS: Plain CSS](https://manikumar.in/blog/mobile-optimized-decap-netlify-cms/)

**Comment by cybardev** [comment]
There's a PR (https://github.com/decaporg/decap-cms/pull/7298) for mobile UI, but sadly it's been stale for a while...
